sdcm/nemesis.py

```diff
@@ -100,7 +100,7 @@ def __init__(self, tester_obj, termination_event):
         self._add_drop_column_target_table = []
         self._add_drop_column_tables_to_ignore = {
             'scylla_bench': '*',  # Ignore scylla-bench tables
-            'a#usertable': '*',  # Ignore alternator tables
+            'alternator_usertable': '*',  # Ignore alternator tables
```

Comment:
so, they dropped the ```#``` from table names?

Reply:
couldn't it be in a external place, and use this string in both cases (here and in common.py), so if it changes again, we will have a single place to update it.

Reply:
yes they dropped it, it was causing too much damage

it's a nice idea to put it into one place, but hopefully it's not gonna change again.